dev-support/hbase-personality.sh

```diff
@@ -140,7 +140,11 @@ function personality_modules
 
   clear_personality_queue
 
-  extra="-DHBasePatchProcess"
+  # Pass maven a -T argument. Should make it run faster. Pass conservative value.
+  # Default is one thread. 0.5C on an apache box of 24 cores and 2 executors should
+  # make for 6 threads? Lets see. See below for more on -T:
```

Comment:
it'll be 12 I think. AFAIK each executor believes there are 24 cores on the box.

Reply:
0.5 * 12 = 6. Let me change the title. It doesn't agree w/ what is in the patch which is trying to keep us close to current settings. Thanks @busbey